share/functions/fish_clipboard_copy.fish

```diff
@@ -2,7 +2,11 @@ function fish_clipboard_copy
     # Copy the current selection, or the entire commandline if that is empty.
     set -l cmdline (commandline --current-selection | string collect)
     test -n "$cmdline"; or set cmdline (commandline | string collect)
-    if type -q pbcopy
+
+    if set -q SSH_TTY; and not set -q DISPLAY
+        set -l encoded (printf %s $cmdline | base64 | string join '')
+        printf '\e]52;c;%s\a' "$encoded" >/dev/tty
```

Comment:
What if `/dev/tty` doesn't exist?

IIRC it's a linuxism?

Reply:
`/dev/tty` isn't available under BSD.

Reply:
Right we can probably write to `$SSH_TTY` instead

actually it also seems to work if when write to stdout, that seems ideal

Reply:
It works writing to stdout unless you do `fish_clipboard_copy | cat`.

Probably just error out if `isatty stdout` fails?

Reply:
> Probably just error out if `isatty stdout` fails?

Ok. Maybe more elegant to write to $SSH_TTY but that might have spooky failures after tmux re-atttach, I'm not sure

Reply:
> `/dev/tty` isn't available under BSD.

Huh. Apparently it's even part of [POSIX](https://pubs.opengroup.org/onlinepubs/000095399/basedefs/xbd_chap10.html) and available on macOS.